vendor/composer/autoload_classmap.php

```diff
@@ -3397,15 +3397,15 @@
     'Zend\\XmlRpc\\Value\\Nil' => $vendorDir . '/zendframework/zendframework/library/Zend/XmlRpc/Value/Nil.php',
     'Zend\\XmlRpc\\Value\\Struct' => $vendorDir . '/zendframework/zendframework/library/Zend/XmlRpc/Value/Struct.php',
     'Zend\\XmlRpc\\Value\\Text' => $vendorDir . '/zendframework/zendframework/library/Zend/XmlRpc/Value/Text.php',
-    'common\\database\\Auditor' => $baseDir . '/common/database/auditor.php',
-    'common\\database\\Connector' => $baseDir . '/common/database/connector.php',
+    'common\\database\\Auditor' => $baseDir . '/common/database/Auditor.php',
+    'common\\database\\Connector' => $baseDir . '/common/database/Connector.php',
```

Comment:
However, that PR should be brought in tomorrow, just have to test @MatthewVita's fix